rules/epmark.go

```diff
@@ -19,9 +19,15 @@ import (
 	"hash/fnv"
 	"io"
 
+	log "github.com/sirupsen/logrus"
+
 	"github.com/projectcalico/felix/markbits"
 )
 
+const (
+	pseudoGenericEndpointName = "caliPseudoGenericEndpoint"
```

Comment:
Please make sure this is an invalid interface name, I think it's enough to make it >32 chars, but you could add a 0-byte in there (which is definitely not valid in an interface name)

Reply:
Done.